app/src/main/java/dev/arkbuilders/arkmemo/media/ArkAudioRecorderImpl.kt

```diff
@@ -47,13 +49,24 @@ class ArkAudioRecorderImpl @Inject constructor(
 
     override fun stop() {
         recorder?.let {
-            it.stop()
+            try {
+                it.stop()
+            } catch (e: RuntimeException) {
+                Log.e(TAG, "stop exception: " + e.message)
+            }
+
```

Comment:
I think we don't need to handle these exceptions as we always call these method after `init()` and `start()`
```suggestion
        recorder?.stop()
        recorder?.release()
```

Reply:
@hieuwu We should still handle the exception because it can go wrong (crash the app) in the future if we mistakenly calling stop on a player when it's not started successfully yet. After all, the app should not crash due to any reason.